roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -18,7 +18,7 @@ ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --restart=always
 {% if etcd_after_v3 %}
 {{ etcd_container_bin_dir }}etcd
 {% endif %}
-ExecStartPre=-{{ docker_bin_dir | default("/usr/bin") }}/docker rm -f {{ etcd_member_name | default("etcd-proxy") }}
+ExecStopPost=-{{ docker_bin_dir | default("/usr/bin") }}/docker rm -f {{ etcd_member_name | default("etcd-proxy") }}
```

Comment:
We have --restart=always set on line 9. If you reboot the host or restart docker (and it stops containers), etcd container will start up automatically. If you want this behavior to work, you need to change line 9 to read like this:

``` ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --rm \```